packages/wagtail/modules/components/Blocks/BannerBlock/__storybook__/mockBannerBlockProps.ts

```diff
@@ -0,0 +1,38 @@
+import type { IBannerBlockProps } from '../types'
+import imageFile from './static/image.jpeg'
+
+export const mockBannerBlockProps: IBannerBlockProps = {
```

Comment:
we moved away from adding a `I` prefix for interfaces, would be cool to do the same on this package